services/ddd.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+export const getDddsData = async () => {
+  const url =
+    'http://www.anatel.gov.br/dadosabertos/PDA/Codigo_Nacional/PGCN.csv';
+  const LINE_BREAK = '\r\n';
+
+  const { data: reponseData } = await axios({
+    url,
+    method: 'get',
+    responseEncoding: 'binary',
+  });
+
+  const body = reponseData.toString('binary');
+
+  const lines = body.split(LINE_BREAK);
+
+  lines.shift();
```

Comment:
Será que um destructuring do header do restante não ficaria bom? 

```suggestion
  const [header, ...lines] = body.split(LINE_BREAK);
```

Reply:
Nossa sensacional essa forma de separar o header das linhas, eu não conhecia, muito obrigado por compartilhar :+1:. 
No meu caso eu acredito que seja melhor deixar com o lines.shift() pois não vou utilizar o header, mas se tiver mais alguma sugestão é só falar.